hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -22,8 +22,6 @@
   // Default file to store build_seed value
   build_seed_file_path: "{build_dir}/build_seed.log"
 
-  // pass and fail patterns
-  build_pass_patterns: []
   // TODO: Add back FuseSoC fail pattern after
   // https://github.com/lowRISC/opentitan/issues/7348 is resolved.
   build_fail_patterns: []
```

Comment:
Maybe remove this too?

Reply:
I didn't remove it because I did not want to look at the Issue #7348 haha.. Let me remove it.

Reply:
Right - the issue is still open and we can fix that separately.